store/tikv/coprocessor.go

```diff
@@ -545,27 +630,33 @@ func (it *copIterator) open(ctx context.Context) {
 				minCommitTSPushed: &it.minCommitTSPushed,
 				Client:            it.store.client,
 			},
-
-			memTracker: it.memTracker,
-
+			memTracker:      it.memTracker,
 			replicaReadSeed: it.replicaReadSeed,
 		}
 		go worker.run(ctx)
 	}
+	bo := NewBackoffer(ctx, copBuildTaskMaxBackoff).WithVars(it.vars)
+	_, err := buildCopTasksChan(bo, it.store.regionCache,
```

Comment:
The `errCh` should not be discarded here, and we should check `errCh` each time it tries to retrieve a task somewhere.